frontend/package.json

```diff
@@ -58,7 +58,8 @@
     "rollup-plugin-postcss": "4.0.2",
     "rollup-plugin-terser": "7.0.2",
     "rollup-plugin-typescript2": "0.31.1",
-    "sprotty": "0.9.0",
+    "sprotty": "0.11.1",
+    "sprotty-protocol": "0.11.0",
```

Comment:
Why the difference in `devDependencies` vs `peerDependencies`?

Reply:
_sprotty-protocol_ is not needed in `peerDependencies` as it comes with Sprotty.
If I don't put _sprotty-protocol_ in `devDependencies`, _sprotty-protocol_ imports work but the auto-completion will not propose _sprotty-protocol_.

Reply:
I have tested on my end while updating the version of sprotty used and it does work. Could you just remove `spotty-protocol` here, I understand the appeal of having the completion support but it won't prevent anyone from typing `import Stuff from 'sporty'` and it could create problems if we don't update it properly by having incompatible versions of sprotty and sprotty-protocol (we already have different version number here). If you prefer it the other way, just make sure that everything is aligned in all the package.json in all the locations.

I don't really mind that you do that one way or the other, I just want consistency in the package.json (for example, in the devDependencies of Sirius Components and the dependencies of Sirius Web). Could you move on with the submission of the other PRs so that I can merge all that and release Sirius Components 2022.01.10, thanks :)